changelog/26777.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:change
+audit: breaking change - Vault now allows audit logs to contain 'correlation-id' and 'x-correlation-id' headers when they 
```

Comment:
❓ What is the user impact given the breaking change?

Reply:
There's a corresponding docs PR that tries to call out the change: https://github.com/hashicorp/vault/pull/26778.

The Vercel deployment currently lives here: https://vault-n5poe4e19-hashicorp.vercel.app/vault/docs/audit#audit-request-headers but could change if there are any more pushes to that branch.

Reply:
Got it, thanks for the clarification!